### DISCUSSION TITLE
Seed Medusa Database More than ones

### DISCUSSION THREAD

**Discussion by Rahul-jain01** [medusajs/medusa#5069] [Q&A]
Initially, I seeded the database with default data from the `seed.json` file. Then, I created new entities and performed a migration, resulting in the creation of new tables in the database. Now, I want to seed the database again with new data according to my requirements.

Scenario i have table SearchHistory  in which I am maintain the customer search history 

 
@Entity()
export class SearchHistory extends BaseEntity {
  @Column({ type: "numeric" })
  count: number | null;

  @Column({ type: "varchar" })
  name: string | null;

  @Column()
  customer_id: string;

  @ManyToOne(() => Customer)
  @JoinColumn({ name: "customer_id" })
  customer: Customer;

  @Column("jsonb", { nullable: true })
  meta_data?: object[];

}
 
 so for that entity can I create the new seed file and seed again

**Comment by pevey** [comment]
Yes, you can create and run multiple seed files.
